intermediate-js.js

```diff
@@ -3,8 +3,11 @@
 // 1. Write an arrow function that takes one argument of type array and returns the contents of that array in reverse order.
 
 var originalArray = [1, 2, 3, 4, 5, 6, 7]
+reverseArray = array => {
```

Comment:
Looks good! Watch your formatting though. Lines 7 and 8 could be tabbed in and you might want to get into the habit of putting your closing brackets on the next line. (ei moving curly on line 8 to line 9)